Source/AliveLibCommon/AnimResources.cpp

```diff
@@ -483,6 +479,10 @@ constexpr CombinedAnimRecord kAnimRecords[] = {
     {AnimId::Slurg_Burst, { "SLURG.BAN", 2808, 46, 15, kSlurgResID, PalId::Default}, kNullAnimDetails },
     {AnimId::Slurg_Turn_Around, { "SLURG.BAN", 2740, 46, 15, kSlurgResID, PalId::Default}, kNullAnimDetails },
 
+    {AnimId::Background_Glukkon, kNullAnimDetails, { "GLUKKON.BND", 46096, 68, 60, kGlkbasicResID, PalId::Default} },
+    {AnimId::Bell_Hammer, kNullAnimDetails, { "TESTANIM.BAN", 4488, 71, 69, kHammerResID, PalId::Default} },
```

Comment:
Think this one seems to be embedded into the camera, but if TESTANIM is the same one then I guess that works ?